README.md

```diff
@@ -253,6 +254,20 @@ Enables debug logging mode.
 _M.px_debug = true
 ```
 
+#### <a name="customblockpage"></a> Custom Block Page
+
+Block page could be customize.
```

Comment:
Small change to below.

`Users can customize the blocking page to meet their branding and message requirements. 
Specify the URL to a blocking page html file. The page can also implement reCaptcha. See <docs location> for more examples of a customized reCaptcha page.`

Can also show an example of handing the reCaptcha similar to how it is implemented in the Apache README.